identify/extensions.py

```diff
@@ -223,21 +225,26 @@
     '.bash_profile': EXTENSIONS['bash'],
     '.bashrc': EXTENSIONS['bash'],
     '.bowerrc': EXTENSIONS['json'] | {'bowerrc'},
+    '.browserslistrc': {'text', 'browserslistrc'},
+    '.codespellrc': EXTENSIONS['ini'] | {'codespellrc'},
     '.coveragerc': EXTENSIONS['ini'] | {'coveragerc'},
     '.cshrc': EXTENSIONS['csh'],
+    '.csslintrc': EXTENSIONS['json'] | {'csslintrc'},
     '.dockerignore': {'text', 'dockerignore'},
     '.editorconfig': {'text', 'editorconfig'},
     '.flake8': EXTENSIONS['ini'] | {'flake8'},
     '.gitattributes': {'text', 'gitattributes'},
     '.gitconfig': EXTENSIONS['ini'] | {'gitconfig'},
     '.gitignore': {'text', 'gitignore'},
+    '.gitlint': EXTENSIONS['ini'] | {'gitlint'},
     '.gitmodules': {'text', 'gitmodules'},
     '.hgrc': EXTENSIONS['ini'] | {'hgrc'},
     '.jshintrc': EXTENSIONS['json'] | {'jshintrc'},
     '.mailmap': {'text', 'mailmap'},
     '.mention-bot': EXTENSIONS['json'] | {'mention-bot'},
     '.npmignore': {'text', 'npmignore'},
     '.pdbrc': EXTENSIONS['py'] | {'pdbrc'},
+    '.pylintrc': EXTENSIONS['ini'] | {'pylintrc'},
```

Comment:
iirc this isn't necessary if there's `pylintrc` -- can you double check this?

Reply:
Also confirmed, dropped this as well.